Sorts/CircleSort.java

```diff
@@ -9,7 +9,6 @@
     @Override
     public <T extends Comparable<T>> T[] sort(T[] array) {
         int n = array.length;
-        while(doSort(array, 0, n - 1));
```

Comment:
What's up with that?

---
**Sema Reaction:** :question: I have a question

Reply:
That's not right, you need a loop that calls doSort.